pennylane/ops/op_math/controlled.py

```diff
@@ -515,14 +515,9 @@ def work_wires(self):
         """Additional wires that can be used in the decomposition. Not modified by the operation."""
         return self.hyperparameters["work_wires"]
 
-    @property
-    def active_wires(self):
```

Comment:
Maybe this should go through the deprecation cycle instead of removed directly? @trbromley

Reply:
Thanks @astralcai, good question! Indeed, our default choice when removing things is to go through a deprecation cycle because we don't want to be surprising users.

However, there are exceptions and I'm happy for this removal to be one:
- `Controlled` is not the typical entrypoint for users, `qml.ctrl()` is.
- `active_wires` feels quite specific and I feel like no user / external dev will have used it.